cmd/operator/deploy/operator/05-deployment.yaml

```diff
@@ -64,6 +67,12 @@ spec:
           runAsUser: 1000
           runAsGroup: 1000
           runAsNonRoot: true
+      priorityClassName: gmp-critical
+      tolerations:
+      - key: "kubernetes.io/arch"
+        operator: "Equal"
+        value: "amd64"
+        effect: "NoSchedule"
```

Comment:
Can we also add the keyless tolerations to the operator? We want to ensure it can always be scheduled similarly with the other resources.
```
      - effect: NoExecute
        operator: Exists
      - effect: NoSchedule
        operator: Exists
```

Reply:
Done.